### ISSUE TITLE
Homebrew install fails

### ISSUE THREAD

**Issue by samrathchadha** [JanDeDobbeleer/oh-my-posh#813]
### Description
Homebrew install fails
```
I◆ ~ ❯❯❯ brew tap jandedobbeleer/oh-my-posh                                                                                          16:52:08
Updating Homebrew...
I◆ ~ ❯❯❯ brew install oh-my-posh                                                                                                     16:52:17
==> Installing oh-my-posh from jandedobbeleer/oh-my-posh
==> Downloading https://github.com/JanDeDobbeleer/oh-my-posh/archive/v3.163.1.tar.gz
==> Downloading from https://codeload.github.com/JanDeDobbeleer/oh-my-posh/tar.gz/v3.163.1
##O#- #
curl: (22) The requested URL returned error: 404
Error: Failed to download resource "oh-my-posh"
Download failed: https://github.com/JanDeDobbeleer/oh-my-posh/archive/v3.163.1.tar.gz
```

**Comment by JanDeDobbeleer** [comment]
Ooh, that release needs to go away as it failed. Will do.

**Comment by JanDeDobbeleer** [comment]
Fixed, although it's back to 3.163.0 in reality.
